test/integration/tags/raw_tag_test.rb

```diff
@@ -24,6 +24,7 @@ def test_open_tag_in_raw
   end
 
   def test_invalid_raw
-    assert_match_syntax_error /tag was never closed/, '{% raw } foo {% endraw %}'
```

Comment:
It ends up hitting the new case and getting the other syntax error, so I edited it to new line at that location.